rust/src/server.rs

```diff
@@ -359,6 +388,11 @@ impl Server {
 
         anyhow::bail!("run_grpc_server exited early");
     }
+
+    fn broadcast(self: &Arc<Self>, inner: EventInner) {
+        // We ignore send errors because they're not relevant to us here
```

Comment:
Can you explain why they're not relevant in this comment as well? The comment still leaves the reason unanswered

Reply:
The only reason a broadcast Sender would return an error is if all the receiver handles are dropped, which is a valid state to be in because they will only exist if StreamEvents is called.